third_party/move/move-compiler-v2/tests/checking/typing/bind_wrong_type.exp

```diff
@@ -1,18 +1,18 @@
 
 Diagnostics:
-error: expected `M::R` but found `M::S` in lvalue
+error: expected `M::S` but found `M::R` in lvalue
```

Comment:
Again, this seems it would be clearer as:

error: assignment to lvalue of type `M::S` expected the same type but found `M::R`

Reply:
Ok, I filed a new issue [to track this.](https://github.com/aptos-labs/aptos-core/issues/9003).